src/ontology/mondo-edit.obo

```diff
@@ -10800,9 +10800,10 @@ is_obsolete: true
 
 [Term]
 id: MONDO:0000912
-name: autosomal recessive nonsyndromic deafness 5
+name: autosomal recessive nonsyndromic hearing loss 5
 def: "An autosomal recessive nonsyndromic deafness that has material basis in variation in the chromosome region 14q12." [DOID:0110507, PMID:8944017]
 synonym: "autosomal recessive deafness 5" EXACT [DOID:0110507]
```

Comment:
I realized that we should change the synonym scope to NARROW for all of the synonyms that have deafness in the name. Would you mind doing this too @matentzn?
thanks!

Reply:
That is a bit separate but yeah, I can do that.

Reply:
THANK YOU! If it is too much to add this to your plate, I can do it manually. Let me know.

Reply:
Ok, for the classes in this branch, under MONDO:0019497, I now made sure that _all_ previously exact synonyms are now NARROW. Double-check though I did it right.